modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -702,6 +702,11 @@ TEST_P(Test_Darknet_layers, mish)
     testDarknetLayer("mish", true);
 }
 
+TEST_P(Test_Darknet_layers, tanh)
+{
+    testDarknetLayer("tanh");
```

Comment:
@aryanshomray 
Could you please update test data to have non-trivial different values in `testdata/dnn/darknet/tanh_out.npy` file?
Current all 8 values are equal to `1`.

---

BTW, GitHub shows "Empty file." in opencv_extra PR, but it is not correct (GitHub bug)

Reply:
Hi, I have made the changes in the data. All 1 values might be because of large input numbers that I had taken for the test.